examples/wordcloud_cn.py

```diff
@@ -60,9 +62,10 @@ def jieba_processing_txt(text):
             mywordlist.append(myword)
     return ' '.join(mywordlist)
 
-
+# the return value of 'jieba_processing_txt' function is words, which have been cleaned
+# collocations=False, the parameter is necessary to reduce the possibility of same words appearing several times
 wc = WordCloud(font_path=font_path, background_color="white", max_words=2000, mask=back_coloring,
-               max_font_size=100, random_state=42, width=1000, height=860, margin=2,)
+               max_font_size=100, random_state=42, width=1000, height=860, margin=2, collocations=False)
```

Comment:
this seems good here.

Reply:
yes, adding 'collocations=False' maybe better.
in the python file, some words is deleted, which maybe cause collocations.
``` python
for myword in liststr.split('/'):
        if not (myword.strip() in f_stop_seg_list) and len(myword.strip()) > 1:
            mywordlist.append(myword)
```